WordPress/twentytwentyfive
# Add theme support for post formats

**Description**
I have re-opened this PR after the discussion in https://github.com/WordPress/twentytwentyfive/issues/17
If we want the theme to have partial support for post formats in 6.7, it needs to be added to the theme before Beta 1, since it is an enhancement. Then if it does not work well enough, it can be removed before the string freeze.

This PR:
- Registers theme support for the formats that are listed in Figma: gallery, quote, link, audio, video
- Adds the HTML archive template for the "link" post format.

**Testing Instructions**

This can be tested with the latest version of Gutenberg active.

1. Set the permalinks on the test installation to anything but plain which does not seem to work.
2. Create one or more posts and assign the "link" post format.
3. View the archive on the front: Add `type/link/`after your test domain in the browser address bar.
9. Confirm that the query loop shows the correct posts.
